File: 2/cpp/Hardware.cpp
#include <string>
using namespace std;

class Hardware{
    private:
        string brand;
        string model;

    public:
        Hardware(){}

        Hardware(string brand, string model){
            this->brand = brand;
            this->model = model;
        }

        void setBrand(string brand)
        {
            this->brand = brand;
        }

        string getBrand()
        {
            return this->brand;
        }

        void setModel(string model)
        {
            this->model = model;
        }

        string getModel()
        {
            return this->model;
        }

        ~Hardware(){}
};

File: 2/java/Hardware.java
public class Hardware{
    private String brand;
    private String model;

    public Hardware(){}

    public Hardware(String brand, String model){
        this.brand = brand;
        this.model = model;
    }

    public void setBrand(String brand)
    {
        this.brand = brand;
    }

    public String getBrand()
    {
        return this.brand;
    }

    public void setModel(String model)
    {
        this.model = model;
    }

    public String getModel()
    {
        return this.model;
    }
}